src/providers/wms/qgswmtsdimensions.h

```diff
@@ -43,7 +42,6 @@ class QgsWmtsDimensions : public QDialog, private Ui::QgsWmtsDimensionsBase
 
     void selectedDimensions( QHash<QString, QString> &dims );
 
-    void on_buttonBox_helpRequested() { QgsContextHelp::run( metaObject()->className() ); }
```

Comment:
I've made a replacement of all `on_buttonBox_helpRequested()` calls to `showHelp()`. Now all help calls to external documentation are consistent.